@kiva/kv-components/vue/KvChip.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+	<div class="tw-inline-flex">
+		<button
+			class="
+			tw-bg-secondary
+			hover:tw-bg-tertiary
+			tw-items-center
+			tw-flex
+			tw-flex-row
+			tw-cursor-pointer
+			tw-whitespace-nowrap
+			tw-rounded
+			tw-h-2
+			tw-py-2
+			tw-px-2
+			tw-mr-2
+			tw-mb-2
+			tw-select-none
+			"
+			@click="handleClick"
+		>
+			<div>
+				<div
+					v-if="!email"
```

Comment:
Thinking of how this component might be used in other contexts I don't think we should name this `email` -- I think a better approach might be to use a default slot. This would allow you to do things like: 
```<KvChip><h4>@email</h4></KvChip>``` or 
```<KvChip>Long Title</KvChip>```
And greatly simplify your component